### PR TITLE
SITES-5925-[Title] Align size config with accordion

### LINKED ISSUE #2142: [Title] Align size config with accordion
Make the UX of the title policy and edit dialogs consistent with the accordion component.

### PR DESCRIPTION
**Pull request by comanV**
Fixes #2142.
Screenshot 2022-04-28 at 19 03 49

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_design_dialog/.content.xml [hunk 1]
```diff
@@ -41,70 +42,60 @@
...
+                                        required="{Boolean}true">
                                         <items jcr:primaryType="nt:unstructured">
                                             <h1
                                                 jcr:primaryType="nt:unstructured"
-                                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                                checked="true"
-                                                name="./allowedTypes"
-                                                text="H1"
+                                                selected="{Boolean}true"
+                                                text="h1"
```
Comment (jckautzmann) [inline content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_design_dialog/.content.xml:58]: Shouldn't it be: H1, H2, ... with capital H?

File: content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml [hunk 2]
```diff
@@ -41,70 +42,60 @@
...
+                                        required="{Boolean}true">
                                         <items jcr:primaryType="nt:unstructured">
                                             <h1
                                                 jcr:primaryType="nt:unstructured"
-                                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                                checked="true"
-                                                name="./allowedTypes"
-                                                text="H1"
+                                                selected="{Boolean}true"
+                                                text="h1"
```
Comment (jckautzmann) [inline content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml:58]: Shouldn't it be: H1, H2, ... with capital H?

Comment (comanV) [reply content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml:58]: I don't think so, we have to change the UX of the title policy dialog to be consistent with the UX of the accordion policy dialog (which utilises small letters).
Screenshot 2022-04-28 at 14 10 09
.

**Review by msagolj — APPROVED** [review]

File: content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml (hunk 2 above)
Comment (comanV) [reply content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml]: @jckautzmann Didn't know about the TOC discussion. Updated here 13d96bdf1c555a16ff9a06c3a5058ebc7ad8610b the edit and design dialog for Accordion, Title and Teaser, to be consistent.

Comment (jckautzmann) [reply content/src/content/jcr_root/apps/core/wcm/components/title/v3/title/_cq_design_dialog/.content.xml]: thx @comanV !

**Pull request merged**